src/main/resources/assets/raisedclouds/lang/zh_cn.json

```diff
@@ -0,0 +1,14 @@
+{
+	"text.autoconfig.raisedclouds.option.baseY": "云层高度",
+	"text.autoconfig.raisedclouds.option.baseY.@Tooltip": "云层的高度，即云层Y轴的值。",
+	"text.autoconfig.raisedclouds.option.cameraAnchor": "摄像机约束",
+	"text.autoconfig.raisedclouds.option.cameraAnchor.@Tooltip[0]": "云层的高度将会随着玩家当前视角的",
```

Comment:
Full Line: `"When the height (Y value) of player current view's camera changes, the (actual/final) cloud height will also follow its change"`

This one is special, I don't think this line's translation is good, but only two line tooltips are not enough for me to explain the actual/final cloud height is (vanilla default base y or override base y) * ( scale ) + ( player view's camera height) , even I really write it in two line, It is too long, I wonder if there is an available solution.